Fatal error: exception Failure("missing return in foo")
